File: Veri_Tipleri.java

public class Veri_Tipleri {

	public static void main(String[] args) 
	{
		byte kucukTamSayi= 30; //1byte yer kaplar, -128 ve 127 arasý sayýlar
		
		char harf = 'B'; //2 byte yer kaplar. 1 haflik karakte taþýmamýza yardýmcý olur.
		char harf2 = 85;
		
		short kisaTamSayi = 102; //2 byte yer kaplar. -32768 ile 32767 arasý
		
		int TamSayi = 2013; //4 byte yer kaplar, Orta dereceli sayýlarý tutar.
		
		long uzunTamSayi = 2000000000; // 8 byte yer kaplar.
		
		float normalKullaným = 4745.96F; // ondalýk 4byte , sonuna F kullanýlmak zorunluluktur
		float bilimselKullaým = 47E4F; 
		
		double normal = 960.7744485102; //8btye ondalýk
		double bilimsel = 96E41;
		
		String karakterKatari = "Java'da String kullanýmý"; // 8byte yer kaplar.
		
		boolean yanlisKarar = false ; // 1byte yer kaplar.
		boolean karar1=(10>4);
		
	}

}

File: Veri Tipleri.cpp
#include<stdio.h>
#include <string.h>
int main()
{
char harf = 'K'; // içerisine girilen harfin Unicode karþýlýðýný tutar
char harf2 = 65;// char 1 byte yer tutar iþaretsiz tam sayýlar niteliði taþýr.

short kisaTamsayi = 55; // 2byte yer kaplar,kýsa tam sayýlarý alýr (-32768 ile 32768 arasý)

int tamSayi= 1993;  // 4 byte yer kaplar, orta dereceli sayýlarý tutar. (-2147483648 ile 2147483648 arasý)

float normalKullanim = 5887.45;
float bilimsekullanim = 5E2; // 4byte yerkaplar. Ondalýklý sayýlardýr.

double normal= 871.9607411; // 8 byte yer kaplar. floattan daha uzun sayýlar alabilir.
double bilimsel = 12E5;

unsigned int isaretsizTamSayi = 188546; // Baþýna unsigned eklediðimizde kapasitesi2 katýna çýkar fakat '-' deðer almaz.

string cumle = "c++'da string kullanýmý" // 8byte yer kaplar. Uzun cümleler yazmamýzý saðlar.

bool dogruKarar = true; // boolean 1 byte yer kaplar. 1-0 True-False iþlemlerinde kullanýlýr.
bool karar1 = (10<4);
}
